docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md

```diff
@@ -0,0 +1,294 @@
+
+### New-AzAdUser
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADUser` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphUser`
+
+- Parameter `ImmutableId` is not supported and has been removed (It is a bug in current preview
+  version. It should be added and equivalent to `OnPremisesImmutableId`)
+
+### Remove-AzAdUser
+
+- Input type of parameter `InputObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADUser` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphUser`
+
+### Update-AzAdUser
+
+- Input type of parameter `InputObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADUser` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphUser`
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADUser` to
+  `System.Boolean`
+
+### Changes of User Object
+
+- `ObjectType` has been replaced by "OdataType`
+
+- `ImmutableId` has been replaced by "OnpremisesImmutableId`
+
+## Group
+
+### Get-AzAdGroup
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+- Parameter `IncludeTotalCount` is not supported and has been removed
+
+### New-AzAdGroup
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+### Remove-AzAdGroup
+
+- Input type of parameter `InputObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+### Changes of Group Object
+
+- `ObjectType` has been replaced by "OdataType`
+
+## Group member
+
+### Get-AzAdGroupMember
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADObject` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphDirectoryObject`
+
+- Parameter `IncludeTotalCount` was removed
+
+- Input type of parameter `GroupObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+### Add-AzAdGroupMember
+
+- Input type of parameter `GroupObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+### Remove-AzAdGroupMember
+
+- Input type of parameter `GroupObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+## Differences between output from AAD and MSGraph
```

Comment:
The content of that section has been merged into previous sections. This whole section should be deleted.